h5p-bildetema/src/components/SearchPage/SearchFilter/SearchFilterDialog.module.scss

```
@@ -0,0 +1,92 @@
+@use "common/abstracts" as *;
+
+.dialog {
+  z-index: 50;
+  position: relative;
```

The font seems to not be the same as the rest of  Bildetema. So, make sure that we set the `font-familiy` to be the same for the dialog and overwrite the default one where it's needed. 

```suggestion
  position: relative;
  font-family: sans-serif;
```